src/role-access/store/reducers/specific-access.reducer.ts

```diff
@@ -19,6 +27,16 @@ export const specificAccessInitialState: SpecificAccessStateData = {
 
 export function specificAccessReducer(currentState = specificAccessInitialState, action: SpecificAccessAction): SpecificAccessStateData {
   switch (action.type) {
+    case SpecificAccessActionTypes.REQUEST_MORE_INFO_SPECIFIC_ACCESS_REQUEST: {
+      return {
+        ...currentState,
+        caseId : action.payload.caseId,
+        requestedRole : action.payload.requestedRole,
+        requestId : action.payload.requestId,
+        jurisdiction : action.payload.jurisdiction,
+        roleCategory : action.payload.roleCategory
```

Comment:
Do we need all this data in here considering some of it is already in the state?